en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+
+### Monetize an API
+
+Let's use the
+[wso2-am-stripe-plugin](https://github.com/wso2-extensions/wso2-am-stripe-plugin/blob/master/src/main/java/org.wso2.apim.monetization/impl/StripeMonetizationImpl.java) to monetize an API.
+
+<html>
+      <div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>The following steps are based on the [Stripe billing engine](https://dashboard.stripe.com/). If you wish to work with another third-party billing engine, you need to first create a custom implementation by extending the <a href="https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/Monetization.java">monetization interface</a>, and only thereafter will you be able to monetize your API.</p>
+      </div> 
+</html>
+
+#### Before you begin
+
+##### (A.) - Configure the billing engine
+
+1.  [Create an account in Stripe](https://dashboard.stripe.com/register) and obtain the keys for the tenant admin.
+    1.  Create an account.  
+
+         Use the details of the tenant admin and verify the account.
+
+    2.  [Obtain the keys](https://stripe.com/docs/keys#api-keys).  
+
+        ![Obtain keys](../../assets/img/Learn/obtain-keys.png)
+        
+2.  Create an account in [Stripe](https://dashboard.stripe.com/register) and obtain the keys for the API provider.  
+    
+     Each API owner should create a Stripe account if they wish to monetize their APIs.
+
+    1.  Create an account.
+        <html>
+        <div class="admonition note">
+        <p class="admonition-title">Note</p>
+        <p>Make sure to configure the <a href="https://dashboard.stripe.com/account">timezone to UTC</a> before creating any objects in the Stripe accounts.</p>
+        </div> 
+        </html>
+
+    2.  [Obtain the keys](https://stripe.com/docs/keys#api-keys).
+
+3.  Create a [connected account](https://stripe.com/docs/connect/quickstart#create-account) in Stripe.  
+    
+     Use the connected account to connect the API provider’s Stripe account to the tenant admin’s Stripe account. After you create the connected account, get the connect ID of the account, and save the details somewhere.
+
+
+##### (B.) - Configure the billing engine with WSO2 API-M
+
+1. Connect WSO2 API Manager to the billing engine.
+
+    1. Download and copy the billing engine related JAR in to the `<API-M_HOME>/repository/components/lib` directory.
+        
+        For Stripe you need to add the [Stripe Java 9.8.0 JAR](https://mvnrepository.com/artifact/com.stripe/stripe-java/9.8.0).
```

Comment:
The directory that the JAR has to be put into is mentioned in the line above and we talk about  Stripe in this line because the example scenario is based on Stripe. However, I reworded the sentence to make sure that users don't have the same questions as you did.